addons/website_slides/views/slide_question_views.xml

```diff
@@ -59,16 +72,29 @@
             <search string="Quizzes">
                 <field name="question"/>
                 <field name="slide_id"/>
+                <filter string="Archived" name="inactive" domain="[('active','=',False)]"/>
```

Comment:
Just for convention (not sure it is in guidelines): we have a filter_ prefix when naming filters (or groupby if it is a grouping filter), and we try to use the field name if possible: filter_inactive is ok, the other one should probably be groupby_slide_id.

It allows to make context keys like search_default_filter_inactive easier to understand when reading it. Same for search_default_groupby_slide_id, we directly know this will do a group by slide_id on the model.